cmd/syft/cli/commands/update.go

```diff
@@ -89,11 +89,12 @@ func isProductionBuild(version string) bool {
 	return true
 }
 
-func fetchLatestApplicationVersion() (*hashiVersion.Version, error) {
+func fetchLatestApplicationVersion(currentVersion string) (*hashiVersion.Version, error) {
 	req, err := http.NewRequest(http.MethodGet, latestAppVersionURL.host+latestAppVersionURL.path, nil)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create request for latest version: %w", err)
 	}
+	req.Header.Add("User-Agent", fmt.Sprintf("%v %v", "Syft", currentVersion))
```

Comment:
I think we should instead modify this function to take the `id clio.Identification` as a parameter (and, by association the `isUpdateAvailable` function), then this line would be something like:
```
req.Header.Add("User-Agent", fmt.Sprintf("%v %v", id.Name, id.Version))
```

Reply:
Thanks for the prompt review! Sounds reasonable to me, I'll try pushing new commits to amend this

Reply:
Done pushing. PTAL again. Thanks!